realm/realm-library/src/main/cpp/java_object_accessor.hpp

```diff
@@ -738,6 +738,16 @@ inline JavaValue from_mixed(realm::Mixed mixed_value){
     }
 }
 
+inline std::vector<Mixed> to_mixed_vector(const std::vector<JavaValue>& java_value_vector) {
```

Comment:
Is this used at all?

Reply:
Yes, it is used by `addAll` and Co. to generate a vector of Mixed in `OsSet` - you'll see it in the other set branches.